drivers/i2c/Kconfig.it8xxx2

```diff
@@ -8,3 +8,17 @@ config I2C_ITE_IT8XXX2
 	  Enable I2C support on it8xxx2_evb.
 	  Supported Speeds: 100kHz, 400kHz and 1MHz.
 	  This driver supports repeated start.
+
+config I2C_ADJUST_SCL_LOW_PRESCALE
```

Comment:
1. Why can't this be in Devicetree?
2. This option has no dependency set on IT8XXX2. By looking at it, it's not clear it's an option specific to IT8XXX2.

Reply:
1. Due to the fact that the logic design of I2C channel 0, 1, 2 is different from the design of channel 3, 4, 5, they are two different logic blocks in the silicon. Hence, the config here only works for adjusting the frequency of I2C3, I2C4, and I2C5. If we add the properties into the devicetree, I’m afraid that users could probably misunderstand that the frequency of I2C0, 1, 2 could be adjusted through it but actually it’d not work and I don’t know if it’d cause side effects. It’s also mentioned in the line 17 of the drivers/i2c/Kconfig.it8xxx2, we made some descriptions on this.
2. Done.